src/lib/storeTypes.ts

```diff
@@ -8,7 +8,15 @@ export type DotApi = {
     wsProvider?: WsProvider,
     keyring?: Keyring,
     options?: any,
+    selectedEndpoint?: string,
 }
 
-export const defaultDotApi: DotApi = {}
-
+export const defaultDotApi: DotApi = {
+    selectedEndpoint: "",
+    options: {},
+}
+export enum ActionForms  {
+    NoForm,
+    AddControlKey,
+    OtherForm,
```

Comment:
I'm just using this crude mechanism to determine what form is showing. We can do the fancy Svelte style component instantiation maybe later.